src/Workspaces/Core/Portable/NamingStyles/Serialization/AccessibilityExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Xml.Linq;
+
+namespace Microsoft.CodeAnalysis.Diagnostics.Analyzers.NamingStyles
+{
+    internal static class AccessibilityExtensions
+    {
+        internal static bool MatchesSymbol(this Accessibility accessibility, ISymbol symbol)
+        {
+            return symbol.DeclaredAccessibility == accessibility;
+        }
+
+        internal static XElement CreateXElement(this Accessibility accessibility)
+        {
+            return new XElement("AccessibilityKind", accessibility);
+        }
+
+        internal static Accessibility FromXElement(XElement accessibilityElement)
+        {
+            return (Accessibility)Enum.Parse(typeof(Accessibility), accessibilityElement.Value);
```

Comment:
We don't do this for code styles, are you saying we should handle this for all cases where we read XML from the option persistence service?  Sounds like a good idea